File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.cc
#include <stdio.h>

int main(void) {
    int n, x; scanf("%d %d", &n, &x);

    int tmp;
    for (int i = 0; i < n; i++) {
        scanf("%d", &tmp);
        if (tmp < x) printf("%d ", tmp);
    }
    return 0;
}

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int x = sc.nextInt();

        int tmp;
        for (int i = 0; i < n; i++) {
            tmp = sc.nextInt();
            if (tmp < x) System.out.print(tmp + " ");
        }
    }
}